### ISSUE TITLE
[regression] hevc encoding issue on TGL

### ISSUE THREAD

**Issue by zhonghongcheng** [Intel-Media-SDK/MediaSDK#2236]
On TGL, the regression started with commit d31b1550. 

Check https://media-ci.ostc.intel.com/build/456332/junit_report/by_test for the error cases.

Here are more details:
1. Do encode, decode and psnr with previous good commit (00253024):
```bash
$ gst-launch-1.0 -vf filesrc location=/opt/media/src/assets/otc-media/sanity/yuv/1080P_1920x1080.P010 num-buffers=150 ! rawvideoparse format=p010-10le width=1920 height=1080 framerate=30 ! videoconvert dither=0 ! video/x-raw,format=P010_10LE ! msdkh265enc rate-control=vbr hardware=true gop-size=30 target-usage=4 num-slices=1 max-vbv-bitrate=7000 bitrate=3500 ref-frames=2 low-power=1 ! video/x-h265,profile=main-10 ! h265parse ! filesink location=/tmp/00253024.h265
$ gst-launch-1.0 -vf filesrc location=/tmp/00253024.h265 ! h265parse ! msdkh265dec hardware=true ! videoconvert dither=0 ! video/x-raw,format=P010_10LE ! checksumsink2 file-checksum=false frame-checksum=false plane-checksum=false dump-output=true qos=false dump-location=/tmp/00253024.yuv

compare 00253024.yuv to 1080P_1920x1080.P010:
Y, U, V psnr_avg: 33.8901, 40.8487, 40.0222
```
1. Do encode, decode and psnr with FBC commit (d31b1550):
```bash
$ gst-launch-1.0 -vf filesrc location=/opt/media/src/assets/otc-media/sanity/yuv/1080P_1920x1080.P010 num-buffers=150 ! rawvideoparse format=p010-10le width=1920 height=1080 framerate=30 ! videoconvert dither=0 ! video/x-raw,format=P010_10LE ! msdkh265enc rate-control=vbr hardware=true gop-size=30 target-usage=4 num-slices=1 max-vbv-bitrate=7000 bitrate=3500 ref-frames=2 low-power=1 ! video/x-h265,profile=main-10 ! h265parse ! filesink location=/tmp/d31b1550.h265
$ gst-launch-1.0 -vf filesrc location=/tmp/d31b1550.h265 ! h265parse ! msdkh265dec hardware=true ! videoconvert dither=0 ! video/x-raw,format=P010_10LE ! checksumsink2 file-checksum=false frame-checksum=false plane-checksum=false dump-output=true qos=false dump-location=/tmp/d31b1550.yuv

compare d31b1550.yuv to 1080P_1920x1080.P010:
Y, U, V psnr_avg: 33.4849, 40.7429, 39.8831
```

**Comment by lakulako** [comment]
There is addition to this commit https://github.com/Intel-Media-SDK/MediaSDK/pull/2229 for LowPower case.
Minimal content dependent quality shift is still possible because of increasing number of reference for LDB frames.
@zhonghongcheng, please check.

**Comment by lakulako** [comment]
@zhonghongcheng please hold on. We need some time for additional exploration.

**Comment by lakulako** [comment]
@zhonghongcheng, #2229 with #2239 should have fixed the issue. Please try.

**Comment by zhonghongcheng** [comment]
> @zhonghongcheng, #2229 with #2239 should have fixed the issue. Please try.

I have tested your commit #2239, the issue is fixed! But It seems that #2239 is in state "Merging is blocked".
